includes/admin/upgrades/class-pum-admin-upgrade-routine-4.php

```diff
@@ -82,7 +82,7 @@ public static function run() {
 				$targeting_conditions = popmake_get_popup_meta_group( 'targeting_condition', $popup->ID );
 
 				if ( empty( $targeting_conditions ) ) {
-					if ( $popup->post_status === 'publish' ) {
+					if ( $popup->post_status === 'publish' ) {  // phpcs:ignore
```

Comment:
@s-acv2 - Gonna let you try this one again without giving you the answer because it is really that simple ;p